packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -96,11 +96,13 @@ export class ReadAlongComponent {
   @Prop() cssUrl?: string;
 
   /**
-   * Toggle the use of assets folder for resolving urls. Defaults to on
-   * to maintain backwards compatibility
+   * Toggle the use of an assets folder. All image paths
+   * will be prefixed with this. Defaults to 'assets'.
+   * .readalong files should just contain base filenames
+   * not the full paths to the images.
    */
 
-  @Prop() useAssetsFolder: boolean = true;
+  @Prop() imageAssetsFolder: string = "assets/";
```

Comment:
make it mutable `@Prop({mutable: true}) imageAssetsFolder: string = "assets/";`
so that you can append the trailing slash if missing